### DISCUSSION TITLE
how to start the tomcat server in linux?

### DISCUSSION THREAD

**Discussion by ivanuccle** [community/community#59827] [Programming Help] [answered]
### Select Topic Area

Question

### Body

i tried to install

   1.yum install -zxvf apache-tomcat-6.0.47.tar.gz then 
   2.  export TOMCAT_HOME=/home/mpatil/softwares/apache-tomcat-6.0.37
   3. [root@localhost mpatil]# echo $TOMCAT_HOME 
      /home/mpatil/softwares/apache-tomcat-7.0.47

But it's not working

**Comment by evertonpassos915** [comment, marked as answer]
The command you have typed is /startup.sh, if you have to start a shell script you have to fire the command as shown below:

$ cd /home/mpatil/softwares/apache-tomcat-7.0.47/bin
$ sh startup.sh 
or 
$ ./startup.sh
